CONTRIBUTING.md

```diff
@@ -38,15 +38,17 @@ Clone the repository to your local machine from the Azure DevOps endpoint.
 
 - Using Powershell, navigate to the '/RetrospectiveExtension.Frontend' folder, run `npm install`. This will download all the dependent packages listed in 'package.json'.
 
-- Copy the file `RetrospectiveExtension.Frontend\config\environment.tsx.template` into `RetrospectiveExtension.Frontend\config\environment.tsx` and update the fields
+- When developing or publishing the extension locally, you need to create a `.env` file at the top level directory of the front end project (where `package.json` lives). In lieu of the `.env` file you can set actual environment variables. The contents of the `.env` file are
```

Comment:
As discussed, creating an .env.template similar to what we do in the backend deploy folder could be helpful.

Reply:
added template file